tests/conftest.py

```diff
@@ -20,6 +28,15 @@
 DUMMY_API_KEY = '1824812581259009ca9981580f8f8a9012409eee'
 
 
+def debug(*args, **kwargs):
+    print("Open files during tests: ")
+    proc = psutil.Process()
+    print(proc.open_files())
```

Comment:
On mac we were getting too many open file errors.  Unfortunately this didn't serve to help as the open files are likely coming from not our process...